src/rgw/rgw_rest_s3.cc

```diff
@@ -4608,8 +4612,10 @@ RGWOp *RGWHandler_REST_Bucket_S3::op_put()
     return new RGWPutBucketPolicy;
   } else if (is_object_lock_op()) {
     return new RGWPutBucketObjectLock_ObjStore_S3;
+#if 0
```

Comment:
why remove this code?

Reply:
@dang @kalebskeithley i'd like to resolve this stuff before we merge this. we shouldn't remove existing functionality, especially if we're planning to merge before reef